mantidproject/mantid
# Fix Qt3Support warnings fo group 7

This is group 7 of #16128.

This fixes the Qt3Support warnings for group 7.
## To test:

The changes in this PR have touched many areas of MantidPlot.  Below is a list of items which should be checked. Areas that need thorough testing:
- Check the labels under File and its submenus
- Check that you can create a surface plot
- Check the Plot3D toolbar entry ( I think that is Format)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
